### PR TITLE
[terra-functional-testing] Inline Documentation in VR Service

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary

Adds some inline documentation int he visual regression server. This is not an exhaustive list, I'm just adding them as I come across. Figured less is more with these types of changes.

### CODE REVIEW

File: packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/virtualScroll.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
-export default function virtualScroll(x, y, remove) {
-  const w = x === 0 ? 0 : -1 * x;
-  const h = y === 0 ? 0 : -1 * y;
+/**
+ * Script to apply a negative 2D-translate transformation method to move the document from its current position
+ * to up and to the right for the specified number of pixels horizontally and vertically.
```
Comment (benbcai) [inline packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/virtualScroll.js]: I wasn't sure what to make of this. "...current position to up and to the right...".  Should it just be "...current position to the specified..."?

**Review by benbcai — APPROVED** [review]

File: packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/virtualScroll.js (hunk 1 above)
Comment (emilyrohrbough) [reply packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/virtualScroll.js]: yeah this one was weird. was trying to caputre that this changes the value to the negative x or y value provided

File: packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/pageHeight.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+/**
+ * Script to set the document height.
+ *
+ * @param {String} height - The height to set the document to.
```
Comment (StephenEsser) [inline packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/pageHeight.js:4]: Is height a string to account for units? IE: `"20px"`

File: packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/scroll.js [hunk 3]
```diff
@@ -1,3 +1,10 @@
+/**
+ * Script to scroll to a particular set of coordinates in the document.
+ *
+ * @param {Number} x - The pixel along the horizontal axis of the document.
+ * @param {Number} y - The pixel along the vertical axis of the document.
+ * @returns null
```
Comment (StephenEsser) [inline packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/scroll.js]: Functions without an explicit return will return `undefined`

`null !== undefined`

File: packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/pageHeight.js (hunk 2 above)
Comment (emilyrohrbough) [reply packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/pageHeight.js:4]: Yeah exactly. It's used in these places:
- https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/makeAreaScreenshot.js#L56
- https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/makeAreaScreenshot.js#L98

**Review by StephenEsser — APPROVED** [review]

**Pull request merged**